plugins/lsp/language-servers/default.nix

```diff
@@ -28,7 +28,7 @@ let
     {
       name = "astro";
       description = "astrols for Astro";
-      package = pkgs.nodePackages."@astrojs/language-server";
+      package = pkgs.astro-language-server;
```

Comment:
How long has `pkgs.astro-language-server` existed? Will this be a breaking change for any users who's `pkgs` is slightly older than ours?

Reply:
Since this exact bump.
We end up on the same question as last time: should we start handling multiple distinct revisions of nixpkgs ?
Here sure, we could set `package = pkgs.astro-language-server or pkgs.nodePackages."@astrojs/language-server";`...

Reply:
My ideal would be to evaluate the `package` options lazily, so that this only affects users that are using the module.

I appreciate if you don't think it's worth the effort, but I think it's more user-friendly to have the fallback; especially given most of our users aren't using our nixpkgs lock - either because they aren't using standalone or because they have a `follows`.

Reply:
Sure, I think here we can go for this minor tweak.

Reply:
I've opened  #2004 for the more general issue

Reply:
Here is the nixpkgs issue for the package: https://github.com/NixOS/nixpkgs/pull/309100
The change was merged 5th, August.

Reply:
I opened a bug report https://github.com/NixOS/nixpkgs/issues/333677